apps/prairielearn/src/tests/groupStudent.test.ts

```
@@ -24,15 +24,18 @@ locals.assessmentsUrl = locals.courseInstanceUrl + '/assessments';
-  before('set authenticated user', async () => {
+  // set authenticated user
+  beforeAll(async () => {
     storedConfig.authUid = config.authUid;
     storedConfig.authName = config.authName;
     storedConfig.authUin = config.authUin;
   });
-  before('set up testing server', helperServer.before(TEST_COURSE_PATH));
-  after('shut down testing server', helperServer.after);
-  after('unset authenticated user', async () => {
+  // set up testing server
+  beforeAll(helperServer.before(TEST_COURSE_PATH));
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeAll duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)